Sources/VSM/ViewState/RenderedViewState.swift

```diff
@@ -131,13 +129,116 @@ public struct RenderedViewState<State> {
     ) -> State {
         get {
             let wrapper = instance[keyPath: storageKeyPath]
-            wrapper
-                .stateDidChangeSubscriber
-                .subscribeOnce(to: wrapper.container.publisher) { [weak instance] newState in
-                    guard let instance else { return }
-                    wrapper.render(instance, newState)
-                }
+            wrapper.projectedValue.startRendering(on: instance)
             return wrapper.wrappedValue
         }
     }
 }
+
+// MARK: - RenderedViewState (StateRendering & Other Conformances)
+
+/// Provides functions for rendering state changes in UIKit views and view controllers
+@available(iOS 14.0, *)
+public protocol StateRendering {
```

Comment:
? - What is the purpose of this protocol?  `RenderedContainer` conforms to it but I don't see the protocol referenced anywhere else?

Reply:
Good question. In the back of my mind I wrote this assuming that I was designing this to be as open as possible... even towards an injectable implementation for `@RenderedViewState`. However, now that you've brought it to my attention, I can't conjure a scenario where it would make sense for the implementation of that property wrapper to be injectable.

Thanks for pointing this out. I'll simplify this.